frontend/packages/component-library/package.json

```diff
@@ -345,6 +361,7 @@
     "@types/react-dom": "^18.3.1",
     "classnames": "^2.5.1",
     "clean-package": "^2.2.0",
+    "dompurify": "^3.2.4",
```

Comment:
You'll also need to define this as a peer dependency to ensure library compatability. As it stands, using the component will likely cause a crash in the consuming application due to a missing reference to the library